docs/install/source-step-by-step.md

```diff
@@ -0,0 +1,344 @@
+---
```

Comment:
The reason why I have chosen the addition to the title "step by step" was, that I wrote two chapters with installation instructions according to the fact, that the recommended way, how to install the backend, is still not fixed. The plan is to have one cookiecutter that does most of what I describe in the step by step chapter. Even when this cookiecutter exists, the "step by step" chapter (naming can be changed, of course) is still valid and of interest for in depth knowledge.

Reply:
This confuses me further. Are these the two chapters you mentioned?

- [Install Plone from its Packages](https://deploy-preview-1274--6-dev-docs-plone-org.netlify.app/install/installation-from-packages.html#)
- [Install Plone backend from its Packages – Step-by-step](https://deploy-preview-1274--6-dev-docs-plone-org.netlify.app/install/installation-backend-from-packages-step-by-step.html)

In the screenshot:

![Screen Shot 2022-07-23 at 3 21 33 AM](https://user-images.githubusercontent.com/102112/180601278-5b49959e-b435-446d-a032-d09b58333c50.png)

1. I click [Install Plone from its Packages](https://deploy-preview-1274--6-dev-docs-plone-org.netlify.app/install/installation-from-packages.html).
2. I see that I need to [Install the Plone backend](https://deploy-preview-1274--6-dev-docs-plone-org.netlify.app/install/installation-from-packages.html#install-source-install-backend-pip-label). Do I click that, or...
3. I see there is a navigation item to [Install Plone backend from its Packages – Step-by-step](https://deploy-preview-1274--6-dev-docs-plone-org.netlify.app/install/installation-backend-from-packages-step-by-step.html#). Do I click this one?

It sounds like your intent is for the new-to-Plone developer to follow the instructions [Install Plone from its Packages](https://deploy-preview-1274--6-dev-docs-plone-org.netlify.app/install/installation-from-packages.html#). Then if they want details of that process, they may refer to [Install Plone backend from its Packages – Step-by-step](https://deploy-preview-1274--6-dev-docs-plone-org.netlify.app/install/installation-backend-from-packages-step-by-step.html). Is that what you intend?

As it stands, the naming of these two files and their introductory content makes me pause with uncertainty of how to proceed.

Reply:
The challenge is to provide installation instructions while one installation tool is still to be written. 
The gap that needs to be filled in the documentation, because the installation tool is not ready, is here:
https://6.dev-docs.plone.org/install/source.html#installation-jump-in-and-enfold-the-beauty
![image](https://user-images.githubusercontent.com/1144737/181010918-e5f6abf8-1386-461d-96b3-ac2bb9c954cc.png)
So I wrote the chapter https://6.dev-docs.plone.org/install/source.html as it would be with an existing cookiecutter for a backend installation (or a backend and frontend installation all in one) and placed the note https://6.dev-docs.plone.org/install/source.html#installation-jump-in-and-enfold-the-beauty for information. And this note also links to the instructions https://6.dev-docs.plone.org/install/source-step-by-step.html, how to install the backend in the mean time until this future tool exists. 

Apart from the current situation of installation tools, the chapter https://6.dev-docs.plone.org/install/source-step-by-step.html could stay in place for detailed information about what this future tool does under the hood.

I hope this makes the construction a little bit clearer. 

 

> This confuses me further. Are these the two chapters you mentioned?
> 
>     * [Install Plone from its Packages](https://deploy-preview-1274--6-dev-docs-plone-org.netlify.app/install/installation-from-packages.html#)
> 
>     * [Install Plone backend from its Packages – Step-by-step](https://deploy-preview-1274--6-dev-docs-plone-org.netlify.app/install/installation-backend-from-packages-step-by-step.html)
> 
> 
> In the screenshot:
> 
> ![Screen Shot 2022-07-23 at 3 21 33 AM](https://user-images.githubusercontent.com/102112/180601278-5b49959e-b435-446d-a032-d09b58333c50.png)
> 

This screenshot shows on the right side the anchor navigation of this main chapter. 
I now read you comment and the docu multiple times and I think the supplementary nature of https://6.dev-docs.plone.org/install/source-step-by-step.html would be more obvious if https://6.dev-docs.plone.org/install/source-step-by-step.html would become a subchapter of the main https://6.dev-docs.plone.org/install/source.html What do you think?

>     1. I click [Install Plone from its Packages](https://deploy-preview-1274--6-dev-docs-plone-org.netlify.app/install/installation-from-packages.html).
> 
>     2. I see that I need to [Install the Plone backend](https://deploy-preview-1274--6-dev-docs-plone-org.netlify.app/install/installation-from-packages.html#install-source-install-backend-pip-label). Do I click that, or...
> 
>     3. I see there is a navigation item to [Install Plone backend from its Packages – Step-by-step](https://deploy-preview-1274--6-dev-docs-plone-org.netlify.app/install/installation-backend-from-packages-step-by-step.html#). Do I click this one?
> 
> 
> It sounds like your intent is for the new-to-Plone developer to follow the instructions [Install Plone from its Packages](https://deploy-preview-1274--6-dev-docs-plone-org.netlify.app/install/installation-from-packages.html#). Then if they want details of that process, they may refer to [Install Plone backend from its Packages – Step-by-step](https://deploy-preview-1274--6-dev-docs-plone-org.netlify.app/install/installation-backend-from-packages-step-by-step.html). Is that what you intend?
> 
> As it stands, the naming of these two files and their introductory content makes me pause with uncertainty of how to proceed.